parsimonious/expressions.py

```diff
@@ -13,6 +13,7 @@
 from parsimonious.nodes import Node, RegexNode
 from parsimonious.utils import StrAndRepr
 from six import integer_types, python_2_unicode_compatible
+from six.moves import range
```

Comment:
We should order imports according to PEP 8, with six in a prior stanza to internal parsimonious imports.